mobile/src/people/employees-list.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { FlatList, Text, View, StyleSheet } from 'react-native';
+import { connect } from 'react-redux';
+
+import { Department } from '../reducers/organization/department.model';
+import { Employee } from '../reducers/organization/employee.model';
+import { EmployeesStore, EmployeeMap, EmployeeIdsGroupMap } from '../reducers/organization/employees.reducer';
+import { AppState } from '../reducers/app.reducer';
+import { EmployeesListItem } from './employees-list-item';
+import { employeesListStyles as styles } from './styles';
+
+interface EmployeesListProps {
+    employees: EmployeeMap;
+    myDepartmentId: string;
+}
+
+interface EmployeesListState {
+    employeesForMyDepartment: Employee[];
+}
+
+const mapStateToProps = (state: AppState): EmployeesListProps => ({
+    employees: state.organization.employees.employeesById,
+    myDepartmentId: state.userInfo.employee ? state.userInfo.employee.departmentId : 'unknown'
+});
+
+class EmployeesListImpl extends React.Component<EmployeesListProps> {
+    public state: {
+        employeesForMyDepartment: Employee[];
+    };
+
+    constructor(props: EmployeesListProps) {
+        super(props);
+        this.state = {
+            employeesForMyDepartment: []
+        };
+    }
+
+    public componentWillReceiveProps(nextProps: Readonly<EmployeesListProps>) {
```

Comment:
No quite. This is logic should be placed in reducer combining with LOAD_EMPLOYEES_FOR_DEPARTMENT action...